src/transformStateWithClones.js

```diff
@@ -8,6 +8,34 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateCopy = { ...state };
+  const statesVers = [];
+
+  for (let i = 0; i < actions.length; i++) {
+    const action = actions[i];
+
+    switch (action.type) {
```

Comment:
Use destructuring for action properies 
const { type, extraData, keysToRemove } = action;
```suggestion
    switch (type) {
```

Reply:
Agree, use desctructuring, check point 5 in checklist